wiki/People/The_Team/Project_Loved_Team/id.md

```diff
@@ -0,0 +1,80 @@
+
+Anggota tim lainnya membantu mengontrol kualitas, pemeliharaan, dan pengorganisasian.
+
+## Anggota Tim
+
+<!-- TODO: should probably document old team members, since they aren't credited anywhere, besides Toy in Trivia -->
+
+![][flag_US] **[clayton](https://osu.ppy.sh/users/3666350)** mengawasi proyek, dan menulis berita dan postingan forum, ![][flag_LT] [huu](https://osu.ppy.sh/users/6044237) mengelola spreadsheet publik dan membantu memoderasi diskusi tentang Loved, dan ![][flag_US] [Noffy](https://osu.ppy.sh/users/1541323), ![][flag_GB] [hypercyte](https://osu.ppy.sh/users/9155377), and ![][flag_US] [eiri-](https://osu.ppy.sh/users/3388410) memeriksa metadata dari beatmap yang akan di pilih melalui pemungutan suara. Anggota tim lainnya adalah kapten:
+
+### Kapten osu!standard
+
+- ![][flag_US] **[DigitalHypno](https://osu.ppy.sh/users/4384207)**
+- ![][flag_US] [BeasttrollMC](https://osu.ppy.sh/users/3171691)
+- ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050)
+- ![][flag_PL] [fartownik](https://osu.ppy.sh/users/56917)
+- ![][flag_GB] [Ventus](https://osu.ppy.sh/users/1986262)
+- ![][flag_LV] [waywern2012](https://osu.ppy.sh/users/5870453)
+
+### Kapten osu!taiko
+
+- ![][flag_CL] **[-Kazu-](https://osu.ppy.sh/users/920861)**
+- ![][flag_IT] [D3kuu](https://osu.ppy.sh/users/7807444)
+- ![][flag_US] [jyake](https://osu.ppy.sh/users/9099822)
+- ![][flag_JP] [nyanmi-1828](https://osu.ppy.sh/users/6866480)
+- ![][flag_KR] [POCARI SWEAT](https://osu.ppy.sh/users/5082685)
+- ![][flag_SE] [Raphalge](https://osu.ppy.sh/users/3918650)
+- ![][flag_RU] [Remus](https://osu.ppy.sh/users/6850949)
+- ![][flag_CL] [Ulqui](https://osu.ppy.sh/users/1263669)
+
+### Kapten osu!catch
+
+- ![][flag_DE] **[Tenshichan](https://osu.ppy.sh/users/1101600)**
+- ![][flag_NL] [Sartan](https://osu.ppy.sh/users/4100941)
+- ![][flag_KR] [Spectator](https://osu.ppy.sh/users/702598)
+- ![][flag_NL] [Wesley](https://osu.ppy.sh/users/2407265)
+- ![][flag_CN] [Yumeno Himiko](https://osu.ppy.sh/users/1806962)
+
+### kapten osu!mania
+
+- ![][flag_US] **[Toaph Daddy](https://osu.ppy.sh/users/7616811)**
+- ![][flag_US] [-mint-](https://osu.ppy.sh/users/8976576)
+- ![][flag_SG] [Abraxos](https://osu.ppy.sh/users/5025064)
+- ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573)
+- ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341)
+
+## Trivia
+
+- ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) membuat tim kapten pada bulan September 2017 bersamaan dengan sistem [Captain's Pick](/wiki/Beatmaps/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) yang selanjutnya diubah namanya menjadi Project Loved. Hingga akhir 2019, dia mengawasi dan mengelola proyek secara keseluruhan.
+
```

Comment:
delete the extra space inbetween bullet points.
```suggestion
```

Reply:
okeeii, thank you!